src/arch/xtensa/hal/mpu.c

```diff
@@ -20,6 +20,8 @@
  * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
@pnx9 sources inside src/arch/xtensa are coming for a 3rd party and we don't modify them. Please remove the patches touching this files.

Also, make sure you use BIT(x) when we really have to deal with a bit of a hardware register. In other cases, this macro can be misleading.